impeller/renderer/backend/gles/pipeline_library_gles.cc

```diff
@@ -212,7 +212,8 @@ std::shared_ptr<PipelineGLES> PipelineLibraryGLES::CreatePipeline(
       desc,          //
       has_cached_program
           ? std::move(cached_program)
-          : std::make_shared<UniqueHandleGLES>(reactor, HandleType::kProgram)));
+          : std::make_shared<UniqueHandleGLES>(UniqueHandleGLES::MakeUntracked(
+                reactor, HandleType::kProgram))));
```

Comment:
I couldn't come up with a test for this, but it will have coverage in in every opengles golden and devicelab test.